_articles/finding-users.md

```diff
@@ -162,4 +160,4 @@ There is no overnight solution to building an audience. Gaining the trust and re
 
 ## Keep at it!
 
-Sometimes, it takes a long time before people notice your open source project. That's okay! Some of the most popular projects today took years to reach high levels of activity. Focus on building relationships instead of a magic bullet. Be patient, and keep sharing your work with those who appreciate it.
+It may take a long time before people notice your open source project. That's okay! Some of the most popular projects today took years to reach high levels of activity. Focus on building relationships in the open source community instead of hoping that your project will miraculously gain popularity. Be patient, and keep sharing your work with those who appreciate it.
```

Comment:
I like the change to "It may take".

I feel the other change makes the idea needlessly verbose. And building relationships doesn't just have to be in the open source community. Building relationships in other venues can help as well and might bring in other audiences that might not have considered open source before.

Reply:
I'll remove "in the open source community" but I'd like to keep the second half because the original phrase, "a magic bullet", was not clear to me. Is that okay with you?